django/contrib/sites/apps.py

```diff
@@ -11,3 +13,4 @@ class SitesConfig(AppConfig):
 
     def ready(self):
         post_migrate.connect(create_default_site, sender=self)
+        checks.register(check_site_id_type, checks.Tags.caches)
```

Comment:
We should add `sites` to `Tags` and document it in [Builtin tags](https://docs.djangoproject.com/en/dev/ref/checks/#builtin-tags).